assets/js/components/settings/SettingsInactiveModules.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * SettingsInactiveModules component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import Data from 'googlesitekit-data';
+import Layout from '../layout/Layout';
+import Notification from '../legacy-notifications/notification';
+import SetupModule from './SetupModule';
+import { Cell, Grid, Row } from '../../material-components';
+import ThumbsUpSVG from '../../../svg/thumbs-up.svg';
+const { useSelect } = Data;
+
+const SettingsInactiveModules = () => {
+	const modulesData = useSelect( ( select ) => select( CORE_MODULES ).getModules() );
```

Comment:
For consistency, let's call this `modules` and below we can  call those `inactiveModules`
```suggestion
	const modules = useSelect( ( select ) => select( CORE_MODULES ).getModules() );
```